docs/en/v0.3/user-guide/write-data/sdk-libraries/go.md

```diff
@@ -23,31 +23,31 @@ then, you can add one `Series` into `Metric`, then create an `InsertRequest` and
 
 ```go
 func Insert() {
-    series := greptime.Series{}              // Create one row of data
-    series.AddStringTag("host", "localhost") // add index column, for query efficiency
-    series.AddFloatField("cpu", 0.90)        // add value column
-    series.AddIntField("memory", 1024)       // add value column
-    series.SetTimestamp(time.Now())          // requird
-
-    metric := greptime.Metric{} // Create a Metric and add the Series
-    metric.AddSeries(series)
-    // metric.SetTimePrecision(time.Second)  // default is Millisecond
-    // metric.SetTimestampAlias("timestamp") // default is 'ts'
-
-
-    // Create an InsertRequest using fluent style
-    // the specified table will be created automatically if it's not exist
-    insertRequest := greptime.InsertRequest{}
-    // if you want to specify another database, you can specify it via: `WithDatabase(database)`
-    insertRequest.WithTable("monitor").WithMetric(metric) // .WithDatabase(database)
-
-    // Fire the real Insert request and Get the affected number of rows
-    n, err := client.Insert(context.Background(), insertRequest)
-    if err != nil {
-        fmt.Printf("fail to insert, err: %+v\n", err)
-        return
-    }
-    fmt.Printf("AffectedRows: %d\n", n)
+	series := greptime.Series{}
+	series.AddStringTag("host", "localhost")
+	series.AddFloatField("cpu", 0.90)
+	series.AddIntField("memory", 1024)
+	series.SetTimestamp(time.Now())
+
+	metric := greptime.Metric{}
+	metric.AddSeries(series)
+
+	insertRequest1 := greptime.InsertRequest{}
+	insertRequest1.WithTable("monitor").WithMetric(metric)
+
+	insertRequest2 := greptime.InsertRequest{}
+	insertRequest2.WithTable("temperature").WithMetric(metric)
```

Comment:
insert the same metric to tables `monitor` and `temperature`, that example isn't good.